src/atomate2/abinit/schemas/mrgddb.py

```diff
@@ -0,0 +1,407 @@
+"""Core definitions of Abinit calculations documents."""
+
```

Comment:
After looking into it, it seems the field ```output``` is the only one that is redundant since it only contains the structure, so I removed it. The directory is present only once and the event_report is not empty because there are abinit comments:
```
No comparison/check is performed for the current and input DDB information 
because argument --nostrict was passed to the command line.
```
but maybe it could also be removed?
```completed_at``` is also present twice but one of them does not take the right time zone into account...